networking/networking_operators/ingress-operator.adoc

```diff
@@ -131,4 +131,5 @@ ifndef::openshift-rosa,openshift-dedicated[]
 == Additional resources
 
 * xref:../../networking/configuring-a-custom-pki.adoc#configuring-a-custom-pki[Configuring a custom PKI]
-endif::openshift-rosa,openshift-dedicated[]
\ No newline at end of file
+endif::openshift-rosa,openshift-dedicated[]
+* link:https://access.redhat.com/solutions/7096271[Viewing the original client source IP through the X-Forwarded-For header in OpenShift Ingress access log]
```

Comment:
```suggestion
* link:https://access.redhat.com/solutions/7096271[Viewing the original client source IP through the X-Forwarded-For header in {product-title} Ingress access log]
```

Reply:
@rameshsahoo111 please update your PR with this suggestion and squash commits if necessary. thank you